CDP4WebServer.IIS/Web.config

```diff
@@ -0,0 +1,68 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
this file will need to be documented on the wiki regarding IIS installation

Reply:
For sure.